algebra/group_power.lean

```diff
@@ -304,6 +304,20 @@ theorem add_monoid.smul_nonneg [ordered_comm_monoid α] {a : α} (H : 0 ≤ a) :
 | 0     := le_refl _
 | (n+1) := add_nonneg' H (add_monoid.smul_nonneg n)
 
+lemma pow_abs [decidable_linear_ordered_comm_ring α] (a : α) (n) : (abs a)^n = abs (a^n) := 
+begin 
+  induction n with n hi,
+  { simp [monoid.pow] },
+  { unfold monoid.pow, rw [hi,abs_mul] }
```

Comment:
Thinking about this proof (and the next one): both induction step should be pure rewrite proofs. 
Can you try to solve both by `simp [*, monoid.pow, abs_mul]`?
The `*` tells the simplifier to use all assumption (i.e. `hi`). When this works then you don't need to give the explicit names to the induction rule and finally you are at the proof
`by induction n; simp [*, abs_mul, monoid.pow]`